packages/common/src/core/types/IHubContent.ts

```diff
@@ -19,7 +19,7 @@ export enum PublisherSource {
  * Data model for content
  */
 export interface IHubContent
-  extends IHubItemEntity,
+  extends Omit<IHubItemEntity, "schemaVersion" | "extent">,
```

Comment:
@tomwayson Note the `Omit<... "extent">` which was needed b/c I added `extent: IExtent` to the `IHubItemEntity`

Reply:
why not have `IHubItemEntity.extent: BBox` so that it aligns w/  `IItem`?

Reply:
Changed to `number[][]` which is what `IItem.extent` is defined as